content/en/docs/releases/next-release-preview/index.md

```diff
@@ -10,3 +10,71 @@ in the next release of Spinnaker. These notes will be prepended to the release
 changelog.
 
 ## Coming Soon in Release 2025.3.0
+
+
+### Spring Boot 3.0.13 Upgrade
+
+As part of the upgrade to Spring Boot 3.0.13, actuator metrics export configuration properties have changed structure.
+
+#### Actuator Metrics Export Property Changes
+
+The metrics export properties have moved from the old schema to the new format:
+
+| Old Property Prefix | New Property Prefix |
+|---------------------|---------------------|
+| `management.metrics.export.<product>` | `management.<product>.metrics.export` |
+
+**Example**
+
+Prometheus configuration keys have changed from:
+
+```properties
+management.metrics.export.prometheus.*
+
+to
+
+management.prometheus.metrics.export.*
+```
+
+### Session data cleanup
+
+Spring Boot 3.0 considers session data cached by Spring Boot 2.7 invalid.  Therefore, users with cached sessions will be unable to log in until the invalid information is removed from the cache. Open browser windows to Spinnaker are unresponsive after the deployment until they’re reloaded.
+Executing:
+
+    $ redis-cli keys "spring:session*" | xargs redis-cli del
+
+on Gate's redis instance removes the cached session information.
+
+
+### YAML Parsing Limits Configuration
+
+**Context:**
+Starting with **SnakeYAML 1.33**, strict safety limits are enforced by default:
+
+* `maxAliasesForCollections = 50`
+* `codePointLimit = 3 145 728`
+
+These limits can cause large or alias-heavy YAML files (such as Kubernetes manifests) to fail during parsing, potentially breaking Spinnaker deployments.
+
+**Improvement:**
+To address this, Spinnaker now exposes two configurable properties that allow operators to override these limits when needed:
+
+```
+spinnaker.yaml.max-aliases-for-collections: <int>   # default: 50
```

Comment:
See https://github.com/spinnaker/spinnaker/pull/7057/files#r2506971594 for potentially changing the config properties.

Reply:
changed it here https://github.com/spinnaker/spinnaker/pull/7057/commits/1d3576312ca81454ffabe32a651b43a7d57d62b4